form.spec

```diff
@@ -0,0 +1,28 @@
+### RPM external form 4.1.033e
+Source: https://gosam.hepforge.org/gosam-installer/form-%{realversion}.tar.gz
+
+%if "%{?cms_cxx:set}" != "set"
+%define cms_cxx g++
+%endif
+
+%prep
+%setup -q -n form-4.1
+
+%build
+
+CXX="$(which %{cms_cxx}) -fPIC"
```

Comment:
looks like it is needed. Its makefile is not adding it resulting in non-position independent code.